packages/client/src/hooks/useAppDispatch/index.ts

```diff
@@ -0,0 +1,4 @@
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from '../../store/types'
+
+export const useAppDispatch = (): AppDispatch => useDispatch<AppDispatch>()
```

Comment:
```suggestion
export const useAppDispatch = (): AppDispatch => useDispatch<AppDispatch>
export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector
```

см https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks